pyuvsim/tests/test_antenna.py

```diff
@@ -44,9 +43,38 @@ def test_get_beam_jones():
 
     vers = pyuvdata.__version__.split('.')
     version = (float(vers[0]), float(vers[1]), float(vers[2]))
-    if version[0] < 2 or (version[0] >= 2 and version[2] < 1):
+    if version[0] < 2 or (version[0] >= 2 and version[2] < 1 and version[1] < 1):
```

Comment:
Won't this line trigger if `version = (3,0,0)`? Should the first part inside the parentheses just be `version[0] == 2`

Reply:
I think you're right... this should be checking for the case that the version is < 2.0.0. Since the tests are running the latest version of pyuvdata, though, we might as well drop this altogether.